src/adt/Recoil_ArrayKeyedMap.js

```diff
@@ -27,6 +27,7 @@ class ArrayKeyedMap<V> {
   constructor(
     existing?: ArrayKeyedMap<V> | Iterable<[mixed, V]>,
   ): ArrayKeyedMap<V> {
+    // $FlowFixMe
```

Comment:
this will result in `unused suppression` warning internally.
can you add `$FlowOSSFixMe` regex to `.flowconfig` and use that instead?

Reply:
Sure